go-selinux/selinux_test.go

```diff
@@ -79,3 +79,28 @@ func TestSELinux(t *testing.T) {
 		t.Log(PidLabel(1))
 	}
 }
+
+func TestCanonicalizeContext(t *testing.T) {
+	if GetEnabled() {
+		con := "system_u:object_r:bin_t:s0:c1,c2,c3"
+		checkcon := "system_u:object_r:bin_t:s0:c1.c3"
+		newcon, err := CanonicalizeContext(con)
+		if err != nil {
+			t.Fatal(err)
+		}
+		if newcon != checkcon {
+			t.Logf("CanonicalizeContext(%s) returned %s expected ", con, newcon, checkcon)
+			t.Fatal("CanonicalizeContext Failed")
+		}
+		con = "system_u:object_r:bin_t:s0:c5,c2"
+		checkcon = "system_u:object_r:bin_t:s0:c2,c5"
+		newcon, err = CanonicalizeContext(con)
+		if err != nil {
+			t.Fatal(err)
+		}
+		if newcon != checkcon {
+			t.Logf("CanonicalizeContext(%s) returned %s expected ", con, newcon, checkcon)
+			t.Fatal("CanonicalizeContext Failed")
```

Comment:
Why we have 2 functions instead of a single one `t.Fatalf()` ? What are the benefits?